recipes/tkwant/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "tkwant" %}
+{% set version = "1.0.0a0" %}
```

Comment:
Is there a fully released, non-alpha version to package?

Reply:
Yes, we plan the full release of the code, documentation and also a supporting paper soon.
However, as the documentation, which includes the installation instructions, is part
of the source code, we like to have an alpha release available in order to better test
and to fine tune our documentation and the code prior to the public release.
This is why the current version is tagged as alpha, and not yet as final 1.0.0 .

Reply:
The "rc" and "dev" are breaking the PEP 440 version scheme https://www.python.org/dev/peps/pep-0440/#pre-releases
that we like to keep. Moreover, to be honest, the CFEP-05 instructions look kind of cumbersome to me.

So at the end we made the final release 1.0.0 and adapted the recipe accordingly.

Reply:
Dear @scopatz, what do you think, is it ok like this or are there some more things to adapt?